LibTest/io/Stdin/first_A02_t02.dart

```diff
@@ -15,6 +15,12 @@
 import "../../../Utils/expect.dart";
 import "dart:io";
 
+Future<String> readOutput(Stream<List<int>> output) async {
+  List<List<int>> result = await output.toList();
+  if (result.isEmpty) return "";
+  return String.fromCharCodes(result.reduce((x, y) => x + y));
```

Comment:
Normally we use e.g.
```dart
return output.transform(io.systemEncoding.decoder).join('');
```
(or `utf8.decoder`)

Which allows us to decode unicode characters as well.